packages/dds/tree/src/test/simple-tree/object.spec.ts

```diff
@@ -261,6 +261,22 @@ const tcs: TestCase[] = [
 		})(),
 		initialTree: { nested: {} },
 	},
+	// Case with explicit stored keys
+	{
+		schema: (() => {
+			const _ = new SchemaFactory("testE");
```

Comment:
I'd prefer not to introduce more of this pattern. Names should be made of descriptive English words, not "_".

Reply:
I erred on the side of matching existing patterns, but I wholeheartedly agree. Will change.